testsuite/smoke-test/src/validator_txns.rs

```diff
@@ -19,13 +19,9 @@ async fn dummy_validator_txns() {
             config.api.failpoints_enabled = true;
         }))
         .with_init_genesis_config(Arc::new(move |conf| {
-            // start with vtxn disabled.
-            conf.consensus_config.disable_validator_txns();
-
-            // start with dkg enabled.
-            let mut features = Features::default();
-            features.enable(FeatureFlag::RECONFIGURE_WITH_DKG);
-            conf.initial_features_override = Some(features);
+            // start with randomness enabled.
+            conf.consensus_config.enable_validator_txns();
```

Comment:
looks like the test changed? from disable vtxn to enable?

Reply:
yes, that's to allow dummy dkg txns through proposal and get executed.

before we were using `DummyTransaction` which does not exist any more.